README.md

```diff
@@ -47,6 +47,24 @@ Put your custom node scripts into a folder called `node` in the root and add sub
 Use `yarn` to install dependencies
 ```yarn install```
 
+## Release
+In order to release a new version of the project.
+
+```yarn release```
+
+A new version will be calculated based on your commit history – make sure to follow [conventional commit message guidelines](https://github.com/conventional-changelog/standard-version#commit-message-convention-at-a-glance).
+Additionally the CHANGELOG.md will be updated automatically.
+
```

Comment:
Would be useful to add an example of `standard-version --first-release`

Reply:
Added this on your behalf :)